samples/functions/csharp/simplechat/Functions.cs

```diff
@@ -90,7 +90,19 @@ public static IActionResult Home([HttpTrigger(AuthorizationLevel.Anonymous)] Htt
                 DataType = request.DataType
             });
 
-            return request.CreateResponse(BinaryData.FromString(new ClientContent("ack").ToString()), WebPubSubDataType.Json);
+            // retrieve counter from states.
+            var states = new ConnectionState(1);
+            var idle = 0.0;
+            if (connectionContext.States != null)
+            {
+                states = JsonConvert.DeserializeObject<ConnectionState>(connectionContext.States["connectionState"] as string);
```

Comment:
the naming `connectionState` sounds too general, how about `CounterState` to make it more specific to the sample

Reply:
`connectionContext.States["connectionState"]` does not exist when first call?

Reply:
connectionContext.States["connectionState"] does not exist with the first run